networkx/algorithms/approximation/tests/test_clique.py

```diff
@@ -1,6 +1,21 @@
-from nose.tools import *
+# test_clique.py - unit tests for the approximation.clique module
+#
+# Copyright 2015 NetworkX developers.
+#
+# This file is part of NetworkX.
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+"""Unit tests for the :mod:`networkx.algorithms.approximation.clique`
```

Comment:
Do tests appear in the documentation? Can I ask what is the purpose of this line?